packages/flame_isolate/lib/flame_isolate.dart

```diff
@@ -0,0 +1,55 @@
+import 'package:flame/components.dart';
+import 'package:integral_isolates/integral_isolates.dart';
+
+export 'package:integral_isolates/integral_isolates.dart';
+
+/// Mixin on [Component] that holds an instance of a long running isolate using
+/// the library integral_isolates.
+///
+/// Using isolate is done by just running [isolate] function the same way you
+/// would run Flutter's compute function.
+///
+/// The following is an example of running a world update cycle when enough time
+/// has passed.
+///
+/// ```dart
+/// class MyGame extends FlameGame with FlameIsolate {
+///   @override
+///   void update(double dt) {
+///     if (shouldRecalculate) {
+///       isolate(recalculateWorld, worldData).then(updateWorld);
+///     }
+///     return super.update(dt);
+///   }
+/// }
+/// ```
```

Comment:
Maybe a short note about the performance impact of adding too many of these here too